contracts/acl/ACL.sol

```diff
@@ -421,7 +424,7 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
 
         // a raw call is required so we can return false if the call reverts, rather than reverting
         bytes memory checkCalldata = abi.encodeWithSelector(sig, _who, _where, _what, _how);
-        uint256 oracleCheckGas = ORACLE_CHECK_GAS;
+        uint256 oracleCheckGas = gasleft().sub(ERROR_GAS_ALLOWANCE);
 
         bool ok;
         assembly {
```

Comment:
Although we've kept this function from having any reverts, I'm wondering if it would be useful to revert if we detect that the oracle used all the available gas.

Reply:
I can add a revert statement here. However, since we just get a `false` return value whether it reverts or is OOG, some of the other tests will have to be modified/removed as they expect it not to revert even if the `staticcall` reverts/returns false.

Reply:
I guess it could give a developer slightly more context as to why a transaction is failing which could be useful.

Reply:
> However, since we just get a false return value whether it reverts or is OOG

We will have to use the `gasleft()` call to detect whether the oracle call ate all the gas (i.e. remaining gas is under `ERROR_GAS_ALLOWANCE`).